### PR TITLE
Remove Kenny from maintainer list

### PR DESCRIPTION
**Pull request by kennyt**
I changed `Current Maintainer` to `Contributor`. If you guys only want maintainers in the file, feel free to remove me

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

**Comment by jfurrow** [comment]
😭

**Comment by mlunoe** [comment]
😭

File: Authors.md [hunk 1]
```diff
@@ -7,10 +7,10 @@ Rafael Corral | rafael@mesosphere.com | Current Maintainer
...
 John Furrow | johnf@mesosphere.com | Current Maintainer
 Chris Sun | christine.sun@mesosphere.com | Current Maintainer
-Kenny Tran | kennyt@mesosphere.com | Current Maintainer
 Mat Appelman | mat@mesosphere.com | Current Maintainer
 Orlando Hohmeier | orlando@mesosphere.com | Current Maintainer
 Pierluigi Cau | pierlo@mesosphere.com | Current Maintainer
 Philipp Hinrichsen | philipph@mesosphere.com | Current Maintainer
 Philip Norman | philip@mesosphere.com | Current Maintainer
 Felix Gertz | felix@mesosphere.com | Current Maintainer
+Kenny Tran | kennyt8tran@gmail.com | Contributor
```
Comment (rcorral) [inline Authors.md]: I'm confused by the `8` in your email address. Do I pronounce it? Is it just there for fun? WHYY ლ(ಠ益ಠლ)

Comment (iamthemuffinman) [inline Authors.md]: Where are you going?!

**Pull request merged**